tools/check-rule-examples.js

```diff
@@ -115,6 +115,17 @@ async function findProblems(filename) {
                 let hasRuleConfigComment = false;
 
                 for (const comment of ast.comments) {
+
+                    if (comment.type === "Block" && /^\s*eslint-env\s/u.test(comment.value)) {
```

Comment:
```suggestion
                    if (comment.type === "Block" && /^\s*eslint-env(?!\S)/u.test(comment.value)) {
```

To match also empty `/*eslint-env*/` comments.